src/shadowbox/docker/run_action.sh

```diff
@@ -35,6 +38,7 @@ declare -a docker_bindings=(
   -v ${SB_CERTIFICATE_FILE}:${SB_CERTIFICATE_FILE}
```

Comment:
See my Dockerfile command. We can significantly simplify this command by tweaking the directory structure.

Reply:
If it's okay with you, I'd rather do this in a separate PR since it would touch the install script. Should be a cleaner change once this PR is merged.